src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -86,6 +86,11 @@ public function createListFromChoices($choices, $value = null)
     {
         if (is_string($value) && !is_callable($value)) {
             $value = new PropertyPath($value);
+        } elseif (is_string($value) && is_callable($value)) {
+            @trigger_error('createListFromChoices() treats callable strings as callable and is deprecated since version 3.1.', E_USER_DEPRECATED);
```

Comment:
@iltar is right. You should change each message to something like :
```php
'Treating strings as callable is deprecated since version 3.1 and will throw an error in 4.0. You should use a "\Closure" instead.'
```